src/Spout/Writer/Style/Style.php

```diff
@@ -302,6 +338,9 @@ public function mergeWith($baseStyle)
         if (!$this->hasSetWrapText && $baseStyle->shouldWrapText()) {
             $mergedStyle->setShouldWrapText();
         }
+        if (!$this->hasSetBackgroundColor && $baseStyle->shouldApplyBackgroundColor()) {
```

Comment:
The default background color is no background color at all ("transparent" I guess). The default background color in my head was only meant to enable setting a background without a parameter.  (```Style::setBackgroundColor()```)  So a developer will have to explicitly "set" a background. Maybe we should drop the "default background" alltogether?